### PR TITLE
New version of ex15

### PR DESCRIPTION
**Pull request by Esteban82**
See #7846 for details.

This would be the new version. It is the same that in the link but I  add  `-Scb -Srl` for `subplot`.
ex15

### CODE REVIEW

**Review by PaulWessel — APPROVED** [review]
Looks good, but a few issues:

1. You have a mix of spaces and tabs for indent and it looks odd depending on how tabs are converted to spaces.  Better to just use tabs as we do elsewhere
2. No need to comment out the "show" - gmtest.in takes care of such things.
3. There is also ex15.bat?
Please fix those before merging since I am approving.

**Comment by Esteban82** [comment]
> but a few issues:

I think that I fixed all of them.

**Review by PaulWessel — APPROVED** [review]

**Comment by Esteban82** [comment]
@PaulWessel could you update the figure within dvc?

**Comment by PaulWessel** [comment]
Yes, tomorrow morning but you need to merge this approved PR into master.  I will do the fig based on the new script.

**Comment by Esteban82** [comment]
Ok, but I can't merge this PR ("some checks haven’t completed yet" and they won't be run since only the documentation was modified).

**Comment by Esteban82** [comment]
In the figure, there is this error:

In the docs it says:

> Our lower left plot illustrates the results of gridding using a nearest neighbor technique ([nearneighbor](https://docs.generic-mapping-tools.org/dev/nearneighbor.html)) which is a local method: No output is given where there are no data.

This is false (see figure). We need to add -N4+m1 to nearneighbor or to modify the text. The default behaviour is "a quadrant search with 100% coverage, i.e., sectors = min_sectors = 4]". So, in the map, only the nodes surrounded by 4 data are highlighted. What should I do? Modify the script or the text?

**Comment by joa-quim** [comment]
Just merged it myself.

**Pull request merged**
